### DISCUSSION TITLE
Issue with SmartMeter LK13BE6067x9

### DISCUSSION THREAD

**Discussion by Folllast** [arendst/Tasmota#19752] [scripter / SML]
I got a SmartMeter LK13BE606739 and use the script from the site 
https://tasmota.github.io/docs/Smart-Meter-Interface/#logarex-lk13be-sml-eg-lk13be6067x9

We have a PV system on the roof, and when we generate more than we consume, the power changes to negative, which is the right thing to do.

The current, however, remains positive, although this should also be negative. Can it be that this script is configured incorrectly, and the sign (-) cuts off?

Thank you very much!

**Comment by MeeThox** [comment]
Ist bei mir genauso .
Mit mein lk13be606759

**Comment by barth11** [comment]
@Folllast  @MeeThox  were you able to find the script which works?
